songs/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>tbfighters.link</title>
+    <meta name="description" content="Redirection server for TBfighters projects.">
+    <meta name="robots" content="noindex, nofollow">
+    <meta name="googlebot" content="noindex, nofollow">
+    <meta property="og:title" content="tbfighters.link">
+    <meta property="og:description" content="Redirection server for TBfighters projects.">
```

Comment:
Can the description be something like "Parody music videos"

Reply:
Done!